submarine-sdk/pysubmarine/tests/models/test_model.py

```diff
@@ -33,10 +36,16 @@ def tearDown(self):
         pass
 
     @pytest.mark.skip(reason="Developing")
-    def test_log_model(self, mocker):
+    def test_log_model(self, mocker, package_type):
```

Comment:
This test is problematic. `package_type` would be None. You should separate different package types into different unit tests.